### PR TITLE
Implement i18n of zh-CN

### PR DESCRIPTION
**Pull request by Bill-Haku**
@xicri Everything is done. Review it in your spare time.

It looks good in my local.

### CODE REVIEW

**Comment by phanect** [comment]
Thanks, I'll check it on my end by next Sunday!

**Comment by xicri** [comment]
@Bill-Haku Thanks, I'll check it on my end by next Sunday!

File: components/hamburger-menu.vue [hunk 1]
```diff
@@ -63,10 +63,10 @@
     "translate": "原神 自動翻訳"
   },
   "zh-CN": {
-    "about": "About",
-    "opendata": "Opendata·API",
-    "history": "History",
-    "translate": "Genshin Machine Translate (Japanese)"
+    "about": "关于",
+    "opendata": "开放数据 · API",
```
Comment (xicri) [inline components/hamburger-menu.vue:67]: Just a confirmation. Is it common in Chinese to add whitespaces around dot (·)？

Comment (Bill-Haku) [reply components/hamburger-menu.vue:67]: Well, both is OK.

Comment (xicri) [reply components/hamburger-menu.vue:67]: OK, then let's keep as is.

**Pull request merged**